File: 8-kyu/thinkful-number-drills-blue-and-red-marbles/thinkful-number-drills-blue-and-red-marbles.c
float guess_blue(int blueStart, int redStart, int bluePulled, int redPulled)
{
  if(blueStart < bluePulled || redStart < redPulled)  return -1;
  int blue = blueStart - bluePulled;
  int red = redStart - redPulled;
  int total = red + blue;
  float probability = (float) blue / (float) total;
  return probability;
}

File: 8-kyu/thinkful-number-drills-blue-and-red-marbles/thinkful-number-drills-blue-and-red-marbles.java
public class BlueAndRedMarbles {
  public static double guessBlue(int blueStart, int redStart, int bluePulled, int redPulled) {
    int blues = blueStart - bluePulled;
    int reds = redStart - redPulled;
    int total = reds + blues;
    
    return (float)blues / total;
  }
}